doc/tasks/brunch.md

```diff
@@ -8,30 +8,50 @@ It lives under the `brunch` namespace and has following configurable parameters:
 parameters:
     tasks:
         brunch:
-            brunch_file: ~
-            task: ~
+            task: build
+            env: production
+            jobs: 4
+            debug: false
             triggered_by: [js, jsx, coffee, ts, less, sass, scss]
 ```
 
 **brunch_file**
```

Comment:
The brunch_file section is removed from the task. 
Is there a reason why it is removed or can it be re-added?
If no, then this section in the documentation should be removed.